Fix KeyError when loading 'Patients by Ward'

DaeronAlagos opened the pull request:
[I57] Patients by Ward page returns a stack trace KeyError related to the next blood glucose diff value. This fix checks to see if the key exists before formatting it and adding it to the 'rec' dictionary.

Commit:
Fix KeyError when loading 'Patients by Ward'

[I57] Patients by Ward page returns a stack trace KeyError related to the next blood glucose diff value. This fix checks to see if the key exists before formatting it and adding it to the 'rec' dictionary.

PeterAlabaster commented on nh_eobs_mental_health/models/nh_clinical_wardboard.py:
```diff
@@ -297,7 +297,13 @@ def read(self, cr, user, ids, fields=None, context=None,
+                    rec['next_blood_glucose_diff'] = rec['next_blood_glucose_diff'] \
+                        if rec['next_blood_glucose_diff'] != '00:00' else ''
+                else:
+                    rec['next_blood_glucose_diff'] = ''
```
This can probably be done better by inverting the check like:
```python
if rec.get('next_blood_glucose_diff') and rec.get('next_blood_glucose_diff') == '00:00':
    rec['next_blood_glucose_diff'] = ''
elif not rec.get('next_blood_glucose_diff'):
    rec['next_blood_glucose_diff'] = ''
```
as I can't see any feasable reason to set `next_blood_glucose_diff` to itself

PeterAlabaster reviewed (changes requested):
tweak to make logic more readable

Commit:
Refactor to not have dict value being explicitly set to itself

[I57] There is no need for the key to be set to it's own value if the value is valid.
